frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -47,8 +47,14 @@ compatible versions of distributable SDK packages to versions of the OpenShift C
 
 | Console Version   | SDK Package                                     | Last Package Version |
 | ----------------- | ----------------------------------------------- | -------------------- |
-| 4.12.x            | `@openshift-console/dynamic-plugin-sdk`         | Latest               |
+| 4.15.x            | `@openshift-console/dynamic-plugin-sdk`         | Latest               |
```

Comment:
Should we put `1.0.0` there directly, since we are going to publish it anyway

Reply:
I was planning to do that post merge of #13566